Plugins/Flow.Launcher.Plugin.BrowserBookmark/Models/CustomBrowser.cs

```diff
@@ -1,45 +1,44 @@
-﻿namespace Flow.Launcher.Plugin.BrowserBookmark.Models
+﻿namespace Flow.Launcher.Plugin.BrowserBookmark.Models;
+
+public class CustomBrowser : BaseModel
 {
-    public class CustomBrowser : BaseModel
-    {
-        private string _name;
-        private string _dataDirectoryPath;
-        private BrowserType browserType = BrowserType.Chromium;
+    private string _name;
+    private string _dataDirectoryPath;
+    private BrowserType _browserType = BrowserType.Chromium;
 
-        public string Name
-        {
-            get => _name;
-            set
-            {
-                _name = value;
-                OnPropertyChanged(nameof(Name));
-            } 
-        }
-        
-        public string DataDirectoryPath
+    public string Name
+    {
+        get => _name;
+        set
         {
-            get => _dataDirectoryPath;
-            set
-            {
-                _dataDirectoryPath = value;
-                OnPropertyChanged(nameof(DataDirectoryPath));
-            }
+            _name = value;
+            OnPropertyChanged();
```

Comment:
`BaseModel` class, which this class extends, has the `[CallerMemberName]` attribute on this argument, so there's no need to pass it manually.